### DISCUSSION TITLE
How to set different resources for code server and its launched runs in k8s?

### DISCUSSION THREAD

**Discussion by gasgallo** [dagster-io/dagster#20143] [Q&A]
Hi,

when deploying with Helm and using the `K8sRunLauncher`, I know that I can configure the pod resources in the following ways:
- set the resources for each code-server deployment and set `includeConfigInLaunchedRuns` to `true` -> launched run uses the resources as defined in the code-server config
- set the resources for each code-server and set `includeConfigInLaunchedRuns` to `false`, set the resources in the `runLauncher` config -> launched run uses the resources as defined in the `runLauncher` config

How can I configure the Helm chart so that the following conditions are satisfied for a multiple code-servers deployment scenario?
- launched runs from different code-servers have different resources configuration
- launched runs have different resources configuration than their own code-server

TL;DR I want to be able to set different resources configuration for launched runs requested by different code-servers, but I don't want the launched run to copy the resources config from the code-server.

Thanks!
